bridgestyle/qgis/expressions.py

```diff
@@ -11,33 +14,33 @@ class UnsupportedExpressionException(Exception):
 OGC_IS_LIKE = "PropertyIsLike"
 OGC_SUB = "Sub"
 
-
-_qbo = QgsExpressionNodeBinaryOperator.BinaryOperator
-binaryOps = {
-    _qbo.boOr: "Or",
-    _qbo.boAnd: "And",
-    _qbo.boEQ: OGC_IS_EQUAL_TO,
-    _qbo.boNE: "PropertyIsNotEqualTo",
-    _qbo.boLE: "PropertyIsLessThanOrEqualTo",
-    _qbo.boGE: "PropertyIsGreaterThanOrEqualTo",
-    _qbo.boLT: "PropertyIsLessThan",
-    _qbo.boGT: "PropertyIsGreaterThan",
-    _qbo.boRegexp: None,
-    _qbo.boLike: OGC_IS_LIKE,
-    _qbo.boNotLike: None,
-    _qbo.boILike: None,
-    _qbo.boNotILike: None,
-    _qbo.boIs: None,
-    _qbo.boIsNot: None,
-    _qbo.boPlus: "Add",
-    _qbo.boMinus: OGC_SUB,
-    _qbo.boMul: "Mul",
-    _qbo.boDiv: "Div",
-    _qbo.boIntDiv: None,
-    _qbo.boMod: None,
-    _qbo.boPow: None,
-    _qbo.boConcat: None,
-}
+if QgsExpressionNodeBinaryOperator is not None:
```

Comment:
Shouldn't `binaryOps` be set to an empty dict if `QgsExpressionNodeBinaryOperator` is `None` to make your tests work? Same for the other QGIS dependencies at other locations?